src/legacy/ui/public/vislib/visualizations/gauges/meter.js

```diff
@@ -21,12 +21,22 @@ import d3 from 'd3';
 import _ from 'lodash';
 import { getHeatmapColors } from '../../components/color/heatmap_color';
 
+const arcAngles = {
+  angleFactor: 0.75,
+  maxAngle: 2 * Math.PI * 1.3,
+  minAngle: 2 * Math.PI * 0.7,
+};
+
+const circleAngles = {
+  angleFactor: 1,
+  maxAngle: 2 * Math.PI,
+  minAngle: 0,
+};
 
 const defaultConfig = {
+  ...arcAngles,
```

Comment:
Seems like this is not used anywhere because the angles are hardcoded from now. Should we clean up the existing visualizations and remove angles from the saved objects? Not sure whether that's important enough.

Reply:
I think you are right!
Unfortunately we don't have any info about usage of these values in config (maybe someone want to have another angle values, who knows 🤔 ).
I would also prefer to get rid of it !